common/src/resource.d.ts__if_typescript

```diff
@@ -1,13 +1,12 @@
 declare module '*.html' {
-  import { IContainer } from '@aurelia/kernel';
-  import { IBindableDescription } from '@aurelia/runtime';
+  import { IContainer, PartialBindableDefinition } from 'aurelia';
```

Comment:
There was some consideration on imports to avoid "aurelia". The individual imports from deeper modules will work for both app and plugin setup.

In future plugin setup, it will likely avoid "aurelia" package. Plugins only depend on the core modules they need, "aurelia" will bring in too many. But this decision is not finalised as far as I knew.

Reply:
@fkleuver can you have a look of the above discussion for the imports?

Reply:
@3cp Sorry for the late reply. But yeah, ideally the imports should be as direct as possible into the original packages. The `'aurelia'` package is there mostly for user ergonomy.

Reply:
What about we keep this change (import only aurelia) for app skeletons? This is easier for app developers to understand.

For future plugin skeletons, I can add a condition in skeletons to import core modules instead.